CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## Current Develop Branch
 
+## 6.7.2 Mon Jul 01 2019
+
+- [#6713](https://github.com/MetaMask/metamask-extension/pull/6713): * Normalize and Validate txParams in TransactionStateManager.addTx too
+- [#6759](https://github.com/MetaMask/metamask-extension/pull/6759): Update to Node.js v10
+- [#Fixes #6694
](https://github.com/MetaMask/metamask-extension/pull/Fixes #6694
): Fixes #6694
+- [#6743](https://github.com/MetaMask/metamask-extension/pull/6743): * Add tests for ImportWithSeedPhrase#parseSeedPhrase
+- [#Fixes #6740](https://github.com/MetaMask/metamask-extension/pull/Fixes #6740): Fixes #6740
+- [#Fixes #6741](https://github.com/MetaMask/metamask-extension/pull/Fixes #6741): Fixes #6741
+- [#6761](https://github.com/MetaMask/metamask-extension/pull/6761): Fixes #6760, correct PropTypes for nextRoute
+- [#6754](https://github.com/MetaMask/metamask-extension/pull/6754): There is currently a bug in chrome that prevents reading source maps

```

Comment:
Oops - this was my commit, it looks like the script took the wrong line as the description because I merged it instead of using `Squash & Merge`. This should be the description:
```suggestion
- [#6754](https://github.com/MetaMask/metamask-extension/pull/6754): Use inline source maps in development
```

Reply:
Actually, there are a few other formatting errors as well. I'll make a commit to update them all.

AFAIK the only place this changelog is published is the GitHub release description - I can update that as well.